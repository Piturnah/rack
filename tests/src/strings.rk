fn main in
  "hello, world!" print print
end
