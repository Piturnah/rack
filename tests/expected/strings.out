----STDOUT----
4198822
13

----STDERR----
